src/compiler/objective_c_generator.cc

```diff
@@ -38,6 +38,15 @@ using ::std::set;
 namespace grpc_objective_c_generator {
 namespace {
 
+inline bool ShouldIncludeMethod(const MethodDescriptor* method) {
```

Comment:
NIT: consider moving this inline util to [objective_c_generator_helpers.h](https://github.com/grpc/grpc/blob/master/src/compiler/objective_c_generator_helpers.h)

Reply:
Done